.github/workflows/ci.yml

```diff
@@ -0,0 +1,101 @@
+name: CI
+
+on:
+  push:
+    # branches:
+    # - main
```

Comment:
This is temporarily commented out so that I can trigger builds on a push to this PR's branch. This should get uncommneted before merge.